### PR TITLE
cel langdef doc links

### PR DESCRIPTION
**Pull request by bitsofinfo**
# Changes

Updated docs to include links to the actual CEL language specification as it is not clear from the docs in their current state what the standard CEL syntax actually is.

### CODE REVIEW

File: docs/eventlisteners.md [hunk 1]
```diff
@@ -241,9 +241,11 @@ spec:
 
 CEL interceptors parse expressions to filter requests based on JSON bodies and
 request headers, using the [CEL](https://github.com/google/cel-go) expression
-language.
+language. Please read the [cel-spec language definition](https://github.com/google/cel-spec/blob/master/doc/langdef.md)
+for more details on the expression language syntax.
 
-Supported features include case-insensitive matching on request headers.
+In addition to the standard CEL expression language syntax, additional
```
Comment (dibyom) [inline docs/eventlisteners.md]: Can we add a link to cel_expressions.md doc here?

Comment (bitsofinfo) [reply docs/eventlisteners.md]: done

**Comment by bitsofinfo** [comment]
can't seem to squash, something jacked in my repo

**Comment by bitsofinfo** [comment]
i think its good now

**Comment by dibyom** [comment]
Thank you!
/lgtm

**Pull request merged**